app/controllers/frontend_log_controller.rb

```diff
@@ -8,26 +8,28 @@ class FrontendLogController < ApplicationController
   # In rare circumstances, these writes can clobber other, more important writes.
   before_action :skip_session_commit
 
+  # Please try to keep this list alphabetical as well!
```

Comment:
This might be an opportunity to script this?

Reply:
We could consider adding a controller spec to check that it's sorted?

e.g.

```rb
expect(described_class::EVENT_MAP.keys).to eq(described_class::EVENT_MAP.keys.sort)
```